modules/client/src/tokenVoting/internal/utils.ts

```diff
@@ -79,21 +79,6 @@ export function toTokenVotingProposal(
       abstain: proposal.abstain ? BigInt(proposal.abstain) : BigInt(0),
     },
     settings: {
-      // TODO
-      // this should be decoded using the number of decimals that we want
-      // right now the encoders/recoders use 2 digit precission but the actual
-      // subgraph values are 18 digits precision. Uncomment below for 2 digits
-      // precision
-
-      // minSupport: decodeRatio(
-      //   BigInt(proposal.totalSupportThresholdPct),
-      //   2,
-      // ),
-      // minTurnout: decodeRatio(
```

Comment:
If the solution comes later, removing this will make things a bit slower

Reply:
My understanding is that this is not going to change, so I dont see a problem. 

The subgraph is settled in using 18 digits and we are ettled in using a number between 0 and 1

Reply:
There is a task on the contract side to standardize the precision numbers (APP-1264)

Reply:
Hmm, ok, nice to know then i'll leave this comment